prow/config/topology/single.json

```diff
@@ -1,10 +1,8 @@
 [
   {
-    "cluster_name": "istio-testing",
-    "pod_subnet": "10.10.0.0/16",
-    "svc_subnet": "10.255.10.0/24",
-    "network_id": 0,
-    "control_plane_index": 0,
-    "config_index": 0
+    "clusterName": "istio-testing",
+    "podSubnet": "10.10.0.0/16",
+    "svcSubnet": "10.255.10.0/24",
+    "network": 0
```

Comment:
network is an int here and string elsewhere. is that intended/ok?

Reply:
Yeah, can we make it match `clusterName`?

Reply:
It should support both since we `jq -r` but I'll make it match.